eventstore/whatsapp_actions.py

```diff
@@ -78,6 +79,30 @@ def handle_event(event):
     if event.is_hsm_error:
         handle_whatsapp_hsm_error(event)
 
+    if event.fallback_channel is True:
+        handle_fallback_delivery_error(event)
+
+
+def handle_fallback_delivery_error(event):
+    try:
+        df = DeliveryFailure.objects.get(contact_id=event.recipient_id)
+        df.number_of_failures += 1
+        df.save()
+
+        if df.number_of_failures >= 5:
+            async_create_flow_start.delay(
+                extra={
+                    "optout_reason": "sms_failure",
+                    "timestamp": event.timestamp.timestamp(),
+                },
+                flow=settings.RAPIDPRO_OPTOUT_FLOW,
+                urns=[f"whatsapp:{event.recipient_id}"],
+            )
+    except DeliveryFailure.DoesNotExist:
+        df = DeliveryFailure(contact_id=event.recipient_id, number_of_failures=0)
+        df.number_of_failures += 1
+        df.save()
```

Comment:
We could just create this with a `number_of_failures` of `1` in the first place, ie.
```
DeliveryFailure.objects.create(contact_id=event.recipient_id, number_of_failures=1)
```